types/index.test.ts

```diff
@@ -3,7 +3,7 @@
 
 // This file will be compiled by multiple versions of TypeScript as decribed in ./test/TypesTests.ts to verify there are no errors
 
-import { AzureFunction, Context, Cookie, HttpMethod, HttpRequest, Timer } from '@azure/functions';
+import { AzureFunction, Context, Cookie, HttpMethod, HttpRequest, HttpResponseApi, HttpResponseObject, Timer } from '@azure/functions';
```

Comment:
I suggest you disable this rule for the whole file:
```
/* eslint-disable @typescript-eslint/no-unused-vars */
```
This file is a bit weird, so some of the rules don't make sense.

Reply:
Done :)